.gitignore

```diff
@@ -15,3 +15,4 @@ MANIFEST
 *.ipynb_checkpoints
 outdir/*
 .idea/*
+.DS_Store
```

Comment:
This is fine for now but we will have to remove this before merging into the main repo.